server/app/views/admin/programs/ProgramPredicateConfigureView.java

```diff
@@ -524,8 +525,23 @@ private DivTag createScalarDropdown(
       }
     }
 
+    if (questionDefinition.isName()) {
```

Comment:
**suggestion:** I meant we could put this above where the address scalar change is happening. So it would be below this code:

```
  if (questionDefinition.isAddress()) {
      scalars = ImmutableSet.of(Scalar.SERVICE_AREA);
      maybeSelectedScalar = Optional.of(Scalar.SERVICE_AREA);
    }
```

I don't think we need to add the isName() function, unless we were using it elsewhere. Instead we could do 

```
if (questionDefinition.getQuestionType().equals(QuestionType.NAME)) {
  scalars = ImmutableSet.of(FIRST, MIDDLE, LAST);
}
```

Does that work or was there a reason it was done this way?